patches/@module-federation__sdk@0.3.2.patch

```diff
@@ -0,0 +1,78 @@
+diff --git a/dist/index.cjs.js b/dist/index.cjs.js
+index 5ac66502f7e3230335fbab7bcf67ecb9614f938f..31ccc081db9aa2b6d20f7a66e5cd67f42f8cc1fe 100644
+--- a/dist/index.cjs.js
++++ b/dist/index.cjs.js
+@@ -604,66 +604,14 @@ function createScriptNode(url, cb, attrs, createScriptHook) {
+             url = hookResult.url;
+         }
+     }
+-    let urlObj;
```

Comment:
No need to patch. You can PR my repo and add a react-native.ts to sdk. Then do whatever you need to do that can detect RN vs node and we can ship it under official.

Reply:
Wanted to showcase what was getting in the way. I would love to remove our custom MF plugin in the future and rely just on MF core 🤞

I can't think of a good way to differentiate RN from other envs though 🤔 one idea I had was to enhance the createScript hook, so we could patch the logic entirely. But I would rather keep the logic in MF altogether like you did with node

Reply:
figured most things out, PR here: https://github.com/module-federation/core/pull/2812